### PR TITLE
Compatible with redis protocol error handlers

### PR DESCRIPTION
**Pull request by ShooterIT**
- Make some errors compatible with redis
- Add max multibulk size limit

Hi @git-hulk I am planting redis tcl regression tests to kvrocks, now, find some protocol errors are different with redis, to make this job easy, we should change this errors, plead have a look.

### CODE REVIEW

**Review by git-hulk — APPROVED** [review]
LGTM, cool. Thanks for your great work!

File: src/redis_request.cc [hunk 1]
```diff
@@ -36,13 +37,17 @@ Status Request::Tokenize(evbuffer *input) {
...
+          if (multi_bulk_len_ > PROTO_MULTI_MAX_SIZE) {
+            free(line);
+            return Status(Status::NotOK, "Protocol error: invalid multibulk length");
           }
           state_ = BulkLen;
         } else {
           if (len > PROTO_INLINE_MAX_SIZE) {
             free(line);
-            return Status(Status::NotOK, "Protocol error: too big inline request");
+            return Status(Status::NotOK, "Protocol error: invalid bulk length");
```
Comment (enjoy-binbin) [inline src/redis_request.cc:50]: so is this change intentional? i see the arg name saying INLINE, just trying to confirm
